app/src/main/res/values/dimens.xml

```diff
@@ -37,9 +38,14 @@
     <dimen name="card_elevation">2dp</dimen>
     <dimen name="card_corner_radius">12dp</dimen>
 
+    <!-- elevation for Plant Detail toolbar - used to convert to pixels when hiding/showing -->
+    <dimen name="toolbar_elevation">5dp</dimen>
+
     <!-- Plant List Header margin -->
     <dimen name="header_margin">24dp</dimen>
 
+    <dimen name="plant_description_min_height">550dp</dimen>
```

Comment:
Add comment explaining that this is to ensure that the top text can be fully scrolled off the screen

Reply:
updated